test/taoensso/faraday/tests/main.clj

```diff
@@ -132,75 +132,81 @@
 
 ;;;; Expressions support
 
-(let [i {:id 10 :name "update me"}]
+(let [i-wrong-name {:id 10 :name "update me"}
+      i-no-name {:id 10}
+      abc-set  {:id 10 :someset #{"a" "b" "c"}}
+      abcde-set {:id 10 :someset #{"a" "b" "c" "d" "e"}}
+      abcdef-set {:id 10 :someset #{"a" "b" "c" "d" "e" "f"}}]
   (after-setup!
    #(far/delete-item *client-opts* ttable {:id 10}))
 
   (expect ; Update expression support in update-item
    {:id 10 :name "foo"}
    (do
+     (far/delete-item *client-opts* ttable {:id 10})
+     (far/put-item *client-opts* ttable i-wrong-name)
      (far/update-item *client-opts* ttable
        {:id 10}
        {:update-expr     "SET #name = :name"
         :expr-attr-names {"#name" "name"}
         :expr-attr-vals  {":name" "foo"}
         :return          :all-new})))
+ 
+  (expect ; Update expression support in update-item check wierd capitalization
+   {:id 10 :biddingEnabled true }
+   (do
+     (far/delete-item *client-opts* ttable {:id 10})
+     (far/put-item *client-opts* ttable  i-no-name)
+     (far/update-item *client-opts* ttable
+       {:id 10}
+       {:update-expr     "SET #fieldName = :value"
+        :expr-attr-names {"#fieldName" "biddingEnabled"}
+        :expr-attr-vals  {":value" true}
+        :return          :all-new})))
 
   (expect ; We can add a set item
-    {:id 10 :name "foo" :someset #{"a" "b" "c"}}
+    abc-set
     (do
+      (far/delete-item *client-opts* ttable {:id 10})
+      (far/put-item *client-opts* ttable  i-no-name)
       (far/update-item *client-opts* ttable
                        {:id 10}
                        {:update-expr     "SET someset = :toset"
                         :expr-attr-vals  {":toset" #{"a" "b" "c"}}
                         :return          :all-new})))
 
   (expect ; We can add update a set
-    {:id 10 :name "foo" :someset #{"a" "b" "c" "d" "e"}}
+    abcde-set
     (do
+      (far/delete-item *client-opts* ttable {:id 10})
+      (far/put-item *client-opts* ttable  abc-set)
       (far/update-item *client-opts* ttable
                        {:id 10}
                        {:update-expr     "ADD someset :toset"
                         :expr-attr-vals  {":toset" #{"d" "e"}}
                         :return          :all-new})))
 
   (expect ; Updating an existing element to a set does not duplicate it
-    {:id 10 :name "foo" :someset #{"a" "b" "c" "d" "e" "f"}}
```

Comment:
It isn't an essential part of the test, so when I duplicated the test, I removed it. 

Tests are not just for testing, they are also for demonstration of an API.

If a name *is* required, that would be good to document somewhere.